apps/src/Sounds.js

```diff
@@ -111,7 +124,7 @@ Sounds.prototype.initializeAudioUnlockState_ = function () {
  */
 Sounds.prototype.isAudioUnlocked = function () {
   // Audio unlock doesn't make sense for the fallback player as used here.
-  return this.audioUnlocked_ || !this.audioContext;
+  return this.audioUnlocked_;
```

Comment:
OK, my attempt at handling this case is in 293b92c48ab91eafdfca261f7dd826fac215fa54. This will set `audioUnlocked_` to `true` when `initializeAudioUnlockState_` throws an exception, or when `window.AudioContext` is undefined (the fallback player case), but will always defer unlocking until after prerendering is finished.